config-samples/sample8.yaml

```diff
@@ -0,0 +1,68 @@
+---
+devices:
+  - board: myboard
+    name: "My Board"
+    description: |
+      My Awesome board
+    console: /dev/ttyABC0
+    fastboot: cacafada
+    ftdi_gpio:
+      description: "s:0x0403:0x6011:FT7YWRL8"
+      interface: A
+      gpios:
+        - power_key:
+            line: 2
+        - fastboot_key:
+            line: 0
+            active_low: true
+        - power:
+            line: 1
+            active_low: true
+  - board: myboard-2
+    name: "My Board 2"
+    description: |
+      My Awesome board Number 2
+    console: /dev/ttyABC1
+    fastboot: cacafada
+    ftdi_gpio:
+      description: "s:0x0403:0x6011:FT456RL8"
+      interface: B
+      gpios:
+        - power:
```

Comment:
After looking at the schema, maybe we should change this to be a hash instead of an array?

Reply:
Yes, it would make it much simpler to describe and check

Reply:
I did the schema and sample changes for both ftdi and local gpio ? if it's ok I'll make the code change for local_gpio and update the ftdi_gpio yaml branch accordingly